From 90970107f3276bfd3fc31436ebcabfe823228b7d Mon Sep 17 00:00:00 2001
From: Sunil Goutham <sgoutham@marvell.com>
Date: Fri, 14 Sep 2018 21:32:37 +0530
Subject: [PATCH 0194/1051] net: octeontx2: Fix IP checksum offload settings

This patch fixes L3 IP checksum enum values.

Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/otx2_struct.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/otx2_struct.h b/drivers/net/ethernet/marvell/octeontx2/otx2_struct.h
index 7ea61ac40341..4a12e359d795 100644
--- a/drivers/net/ethernet/marvell/octeontx2/otx2_struct.h
+++ b/drivers/net/ethernet/marvell/octeontx2/otx2_struct.h
@@ -30,10 +30,10 @@ enum nix_send_ldtype {
 
 /* CSUM offload */
 enum nix_sendl3type {
-	NIX_SENDL3TYPE_NONE,
-	NIX_SENDL3TYPE_IP4,
-	NIX_SENDL3TYPE_IP4_CKSUM,
-	NIX_SENDL3TYPE_IP6,
+	NIX_SENDL3TYPE_NONE = 0x0,
+	NIX_SENDL3TYPE_IP4  = 0x2,
+	NIX_SENDL3TYPE_IP4_CKSUM = 0x3,
+	NIX_SENDL3TYPE_IP6 = 0x4,
 };
 
 enum nix_sendl4type {
-- 
2.17.1

